### PR TITLE
Bell/caching support

### PR DESCRIPTION
**Pull request by songbell**

### CODE REVIEW

File: src/plugins/auto/plugin.cpp [hunk 1]
```diff
@@ -266,12 +266,6 @@ InferenceEngine::Parameter MultiDeviceInferencePlugin::GetMetric(const std::stri
     }
 }
 
-// Is called only when caching is enabled
-IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetwork(const std::string& modelPath,
-                                                                        const std::map<std::string, std::string>& config) {
-    return LoadNetworkImpl(modelPath, {}, config);
-}
-
 IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadExeNetworkImpl(const CNNNetwork &network,
```
Comment (songbell) [inline src/plugins/auto/plugin.cpp:269]: @myshevts maxim, can you help double confirm if this is still needed?
From my understanding, there is loadfromcache implemented both for loadnetwork with modelpath & cnnNetwork.
https://github.com/openvinotoolkit/openvino/blob/master/src/inference/src/ie_core.cpp#L693

Comment (myshevts) [reply src/plugins/auto/plugin.cpp:269]: these lines were added by @nosovmik , so let's ask him

**Comment by myshevts** [comment]
can we add tests, @nosovmik  is there any tests that we can instantiate to make sure the AUTO supports the model caching

File: src/plugins/auto/plugin.cpp (hunk 1 above)
Comment (songbell) [reply src/plugins/auto/plugin.cpp:269]: ah..I suppose the cache optimize is for 2 parts, part A:  modelfile->CNNnetwork and part B: load from cache (modelfile and cnn network both supported).  with these lines, it's expected to support partA + part B, while without , only part B is supported, is my understanding right? @nosovmik

**Pull request closed without merging**